### ISSUE TITLE
Update secrets path

### ISSUE THREAD

**Issue by LucasRoesler** [alexellis/derek#59]
In a conversation with @alexellis  we saw an issue in Derek using the latest master version of openfaas.

```
failed to report status github.RepoStatus{State:"success", TargetURL:"http://<domain>:8080/function/alexellis-hallo", Description:"function successfully deployed as: alexellis-hallo", Context:"DEPLOY"}, error: unable to read private key path: /run/secrets/derek-private-key, error: open /run/secrets/derek-private-key: no such file or directory
```

This is due to openfaas/faas#692

## Possible Solution

Update the path to search `/var/openfaas/secrets`

**Comment by alexellis** [comment]
Thanks @rgee0 for working on this 👍

**Comment by alexellis** [comment]
Marking as high priority due to need to enable testing for developers using newer OpenFaaS versions.

**Comment by rgee0** [comment]
Derek close: Addressed in #69
